app/lib/features/main/widgets/bottom_navigation_widget.dart

```diff
@@ -41,7 +41,7 @@ class BottomNavigationWidget extends ConsumerWidget {
                 bottomNavNar(ref),
                 AnimatedContainer(
                   duration: const Duration(milliseconds: 300),
-                  height: showQuickActions ? 180 : 0,
+                  height: showQuickActions ? 250 : 0,
```

Comment:
Also having this hardcoded means it might not be as responsive as we'd like it to be...

Reply:
This is known thing to manage slide-up and slide-down animation. And checked this in Android, iOS and Desktop platforms with this height value.